ui/design-system-components/src/components/EditProfile/GeneralForm/index.tsx

```diff
@@ -83,16 +83,20 @@ export const GeneralForm: React.FC<GeneralFormProps> = ({
           control={control}
           name="name"
           render={({ field: { name, value, onChange, ref } }) => (
-            <TextField
-              type="text"
-              name={name}
-              label={nameField.label}
-              value={value}
-              onChange={onChange}
-              inputRef={ref}
-            />
+            <>
+              <TextField
+                type="text"
+                name={name}
+                label={nameField.label}
+                value={value}
+                onChange={onChange}
+                inputRef={ref}
+              />
+              {errors.name && <p className="text-sm text-red-600 mt-1">{errors.name.message}</p>}
+            </>
```

Comment:
```suggestion
              <TextField
                type="text"
                name={name}
                label={nameField.label}
                value={value}
                caption={errors.name ? errors.name.message: null}
                status={errors.name ? 'error' : null}
                onChange={onChange}
                inputRef={ref}
              />
```

We already have the props to display error on `TextField`.

Reply:
I didn't notice it :). Will check